contracts/strategy/StrategyStrictBase.sol

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "../openzeppelin/SafeERC20.sol";
+import "../openzeppelin/Math.sol";
+import "../interfaces/IStrategyV2.sol";
+import "../interfaces/IForwarder.sol";
+import "../interfaces/IERC4626Strict.sol";
+import "../interfaces/IStrategyStrict.sol";
+import "../tools/TetuERC165.sol";
+
+/// @title Abstract contract for base strict strategy functionality
+/// @author AlehNat
+abstract contract StrategyStrictBase is IStrategyStrict, TetuERC165 {
+  using SafeERC20 for IERC20;
+
+  // *************************************************************
+  //                        CONSTANTS
+  // *************************************************************
+
+  /// @dev Version of this contract. Adjust manually on each code modification.
+  string public constant STRATEGY_BASE_VERSION = "1.0.0";
+  /// @dev Denominator for compound ratio
+  uint internal constant COMPOUND_DENOMINATOR = 100_000;
+  /// @dev Denominator for fee calculation.
+  uint internal constant FEE_DENOMINATOR = 100_000;
```

Comment:
we do not have fees in strict vaults, you can remove

Reply:
Removed